module-services/service-cellular/ServiceCellular.cpp

```diff
@@ -1474,7 +1474,8 @@ bool ServiceCellular::transmitDtmfTone(uint32_t digit)
     at::Result resp;
     if (channel) {
         auto command           = at::factory(at::AT::QLDTMF);
-        std::string dtmfString = "\"" + std::string(1, digit) + "\"";
+        std::string dtmfString = "\"" + utils::singleDigitToString(digit) + "\"";
```

Comment:
it's a shame that we do not have dedicated parser to this response

Reply:
tbh I didn't want to save the world, only fix DTMF issue :P